perun-cli/removeNotifTemplateMessage

```diff
@@ -8,13 +8,13 @@ use Perun::Common qw(printMessage);
 
 sub help {
 	return qq{
-Removes NotifTemplateMessage from db by id. Id is required field.
-------------------------------------
-Available options:
- --NotifTemplateMessageId        | -i id of the NotifTemplateMessage
- --batch                         | -b batch
- --help                          | -h prints this help
- 
+	Removes NotifTemplateMessage from db by id. Id is required field.
+	------------------------------------
+	Available options:
+	--NotifTemplateMessageId        | -i id of the NotifTemplateMessage
+	--batch                         | -b batch
+	--help                          | -h prints this help
```

Comment:
This change (even if it is in commit message) is just totally different from purpose of this commit so I would prefer to separate it from this one.

Reply:
The same for the other similar changes in this commit which fixes indentation in different scripts.

Reply:
I agree with @stavamichal 

BTW, do you know this indent will be visible in the output? If you want to prevent that either do not indent here or use proper quoting. Example follows:
```perl
sub help {
    return "Removes NotifTemplateMessage from db by id. Id is required field.\n" .
           "------------------------------------\n";
```

Reply:
@stavamichal I moved it to own commit. @licehammer yes, I know indentation is visible and it's like this in all our CLI tools (help is then more readable). Unless we decide to change it globally, I would prefer keeping my change to this tool.